_data/toc.yaml

```diff
@@ -157,6 +155,10 @@ guides:
       title: Manage data in containers
   - path: /engine/docker-overview/
     title: Docker overview
+  - path: /release-notes/docker-ce/
```

Comment:
Yup. I only care about release notes if I'm already using the product, not when I'm getting started.